### PR TITLE
Add physical_quantities jupyter notebook

### PR DESCRIPTION
**Pull request by jreichenbach-msu**
This updates the physical quantities to a notebook. 

 - [x] update to new `plasma` instead of `plasma_array`
 - [x] update to new yml file
 - [x] update attributes for output cells to print data
 - [x] fix warnings in output cells

### CODE REVIEW

**Comment by xiangyun19** [comment]
Dear all,

Thank you very for you kind help. 
However, I still can not sovle my problem.

Since I can not use the 'git clone' for internet constraints, so I installed tardis from source in a --offline mode, and then I downloaded the tardis refdata from https://github.com/tardis-sn/tardis-refdata manually. But I do not know where to put these ref-data. When I ran tardis in the Anaconda envs, I just put the 'tardis_example.yml' and 'kurucz_cd23_chianti_H_He.h5' in the same directory. And I just ran the command 

tardis tardis_example.yml outfile.dat

However, I got the following errors:

File "H5F.c", line 586, in H5Fopen
      unable to open file
File "H5Fint.c", line 1384, in H5F_open
      unable to read superblock
File "H5Fsuper.c", line 360, in H5F__super_read
      file signature not found

Unable to open/create file "kurucz_cd23_chianti_H_He.h5"

And then, I have quickstart.ipynb, and I found another method to run tardis_example.yml:

from tardis.io.atom_data.util import download_atom_data
download_atom_data('kurucz_cd23_chianti_H_He.h5')

However, I got another error:

Downloading atomic data from https://media.githubusercontent.com/media/tardis-sn/tardis-refdata/master/atom_data/kurucz_cd23_chianti_H_He.h5 to home
OpenSSL.SSL.SysCallError:(104, 'ECONNRESET')

I have tried to open the given url with the browser, but failed. I can not visit the website. 

I think, the refdata from https://github.com/tardis-sn/tardis-refdata may not the up to date files, and they are not suit for the current tardis any more. But I can not download the latest ref-data on line. I wonder if you can send me a copy of the latest ref-data, so that I can figure out what the problem is. 
Thank you. I am looking forward to your reply.

Cheers,
Xiangyun Zeng


13207224039@163.com
 
From: jreichenbach-msu
Date: 2019-11-25 08:55
To: tardis-sn/tardis
CC: Subscribed
Subject: [tardis-sn/tardis] Add physical_quantities jupyter notebook (#990)


You can view, comment on, or merge this pull request online at:
  https://github.com/tardis-sn/tardis/pull/990
Commit Summary
Add initial notebook
Add physical_quantities jupyter notebook
File Changes
A docs/physics/physical_quantities.ipynb (414) 
Patch Links:
https://github.com/tardis-sn/tardis/pull/990.patch
https://github.com/tardis-sn/tardis/pull/990.diff

File: docs/physics/physical_quantities.ipynb [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+   "execution_count": 23,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[\u001b[1mtardis.io.config_reader\u001b[0m][\u001b[1;33mCRITICAL\u001b[0m]  No config file named: my_config.yml (\u001b[1mconfig_reader.py\u001b[0m:236)\n"
```
Comment (wkerzendorf) [inline docs/physics/physical_quantities.ipynb]: you need to change this to a file that does exist

**Comment by wkerzendorf** [comment]
@jreichenbach-msu there are several warnings in the output cells - can you look into them.

**Comment by wkerzendorf** [comment]
@jreichenbach-msu please rebase this branch.

**Pull request closed without merging**